src/pages/Reviews.js

```diff
@@ -27,34 +27,33 @@ const Reviews = ({ productId }) => {
     rating: '',
     title: '',
     comment: '',
-    product: productId,
-    user,
+    product: productId
   });
 
   // Get all reviews
   useEffect(() => {
     getAllReviews(productId);
   }, [productId]);
 
-  const getAllReviews = async ({ productId }) => {
```

Comment:
By putting braces around this, Javascript was expecting that other code would be passing an object here (eg: `getAllReviews({ productId: 123 })` ). But looking through this file, I only ever saw it being called with the productId directly (eg: `getAllRevies(123)`). So we should update the function parameter so that we're properly accssing the argument, otherwise we would just get undefined (which is why the reviews were not showing up; they were all being filtered out).